src/main/java/com/smartcar/sdk/AuthClient.java

```diff
@@ -250,7 +250,7 @@ public AuthUrlBuilder authUrlBuilder() {
    *
    * @throws SmartcarException if the API request fails
    */
-  private Auth call(RequestBody requestBody) throws SmartcarException {
+  private Auth call(RequestBody requestBody) throws SmartcarException, SmartcarExceptionV2 {
```

Comment:
This won't return a V2 exception right? Auth endpoints don't have support for that